core/templates/pages/android-page/android-page.component.ts

```diff
@@ -135,6 +143,30 @@ export class AndroidPageComponent implements OnInit, OnDestroy {
     this.featuresShown = featureNumber;
   }
 
+  validateEmailAddress(): boolean {
+    // A simple check for basic email validation.
+    let regex = new RegExp(AppConstants.EMAIL_REGEX);
+    return regex.test(this.emailAddress);
+  }
+
+  subscribeToAndroidList(): void {
+    this.androidUpdatesBackendApiService.subscribeUserToAndroidList(
+      this.emailAddress, this.name
+    ).then((status) => {
+      if (status) {
+        this.alertsService.addInfoMessage('Done!', 1000);
+      } else {
+        this.alertsService.addInfoMessage(
+          'Sorry, some unexpected occurred. Please email admin@oppia.org to ' +
+          'add you to the mailing list.', 10000);
+      }
+    }).catch(errorResponse => {
+      this.alertsService.addInfoMessage(
+        'Sorry, some unexpected occurred. Please email admin@oppia.org to ' +
+        'add you to the mailing list.', 10000);
```

Comment:
Should this message be internationalized (because its on a general learner-facing page)?

Reply:
Gnereally, I haven't seen even learner facing alertsservice messages be translated, that's why I left it (eg: learner dashboard).

Reply:
Ah okay!